config/crd/bases/controlplane.cluster.x-k8s.io_k3kcontrolplanes.yaml

```
@@ -136,7 +136,7 @@ spec:
               hostKubeconfig:
                 description: |-
                   HostKubeconfig is the location of the kubeconfig to the host cluster that the K3k cluster should install in.
-                  Optional, if not supplied the K3k cluster will be made in the current cluster.
+                  If not supplied the K3k cluster will be made in the current cluster.
```

maybe grammatically it can be more clearer this way:
```suggestion
                HostKubeconfig is the location of the kubeconfig for the host cluster where the K3k cluster should be installed. If not supplied the K3k cluster will be made in the current cluster.
```